pxr/usd/usdGeom/mesh.h

```diff
@@ -355,23 +355,23 @@ class UsdGeomMesh : public UsdGeomPointBased
     // --------------------------------------------------------------------- //
     // FACEVARYINGLINEARINTERPOLATION 
     // --------------------------------------------------------------------- //
-    /// Specifies how elements of a _faceVarying_ primvar are
-    /// interpolated. Interpolation can be as smooth as a _vertex_ primvar or
+    /// Specifies how elements of a "faceVarying" primvar are
```

Comment:
In the intro, you used single-quotes for 'points' etc... so why not single-quotes here, too?

Reply:
The single-quotes were to introduce a general term.  Here the references is to specific token names for the primvar interpolation type -- and double-quotes are used for token names.  I can see that using it as an adjective might obscure that...  Perhaps "how elements of a primvar of interpolation type "faceVarying"..." is warranted...

Reply:
Rephrased as suggested in subsequent commit